applications/StructuralMechanicsApplication/custom_elements/base_solid_element.cpp

```diff
@@ -667,9 +675,7 @@ void BaseSolidElement::CalculateOnIntegrationPoints(
         
         // Set constitutive law flags:
         Flags &ConstitutiveLawOptions=Values.GetOptions();
-        ConstitutiveLawOptions.Set(ConstitutiveLaw::USE_ELEMENT_PROVIDED_STRAIN, true);
-        ConstitutiveLawOptions.Set(ConstitutiveLaw::COMPUTE_STRESS, false);
-        ConstitutiveLawOptions.Set(ConstitutiveLaw::COMPUTE_CONSTITUTIVE_TENSOR, false);
+        SetConstituveLawFlags(ConstitutiveLawOptions);
```

Comment:
I think the arguments are wrong here, shouldn't it be `SetConstituveLawFlags(ConstitutiveLawOptions, false);` ?